libraries/AP_Notify/OreoLED_I2C.h

```diff
@@ -19,6 +19,10 @@
 #include <AP_HAL/AP_HAL.h>
 #include "NotifyDevice.h"
 
+#ifndef HAL_WITH_OREO_LED
```

Comment:
I think our pattern would be
```suggestion
#ifndef HAL_OREO_LED_ENABLED
```

ATM.  Similarly elsewhere.  97 of the former, 52 of the latter in some bodgy grepping.  Closer than I thought.

Reply:
Ok I have gone with ENABLED